.github/workflows/push-master.yml

```diff
@@ -1,67 +1,80 @@
-
-
-name: V2LY Merge to master
+name: V2LY master
 
 on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - master
 
 jobs:
-  test:
-    name: Full test suite
-    runs-on: ubuntu-latest
-    steps:
-    - name: Check out
-      uses: actions/checkout@v2
-    - name: Use Node 12
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
-        mongoms_download_mirror: http://downloads.mongodb.org
-        mongoms_version: 4.0.5
-        mongoms_disable_postinstall: 1
-        ci_node_index: 0
-        ci_node_total: 1
-    - name: Full test suite
-      run: |
-        npm ci
-        npm test
-
   docker-build:
     name: Build and push docker container
     runs-on: ubuntu-latest
```

Comment:
probably not important but ubuntu is pretty big - for a docker build there is probably a minimal Linux like Alpine that will do the job.

Reply:
haha it automatically says I "dismissed" a review! I didn't, but we covered this on call.